modules/ROOT/pages/oauth-dance-about.adoc

```diff
@@ -5,18 +5,18 @@ endif::[]
 
 The authentication process performed by the Mule OAuth 2.0 provider, API, and client application conforms to RFC 6749. This process is called the OAuth Dance.
 
-image::oauth-policy1.png[oauth+policy1,height=366,width=640]
+image::mule-oauth-provider-dance.png[oauth+policy1,height=366,width=640]
 
-. A token is requested to the provider, specifying client application credentials as needed (the credentials correspond to a Client ID and secret generated by API Platform).
-. Mule OAuth 2.0 provider returns a token.
-. An HTTP request is performed against the protected resource (ie resource where the OAuth 2.0 Access Token Enforcement policy is applied), including the token either as an authentication header or a query parameter.
+1. A token is requested to the provider, specifying client application credentials as needed (the credentials correspond to a Client ID and secret generated by API Platform).
+2. Mule OAuth 2.0 provider returns a token.
+3. An HTTP request is performed against the protected resource (ie resource where the OAuth 2.0 Access Token Enforcement policy is applied), including the token either as an authentication header or a query parameter.
 - The protected resource is linked with the API definition through Autodiscovery.
```

Comment:
`- The protected resource MUST be linked with the API definition through Autodiscovery.`??

Reply:
No autodiscovery -> No policy applied